darwin/dataset.py

```diff
@@ -98,11 +98,13 @@ def _add_files_to_dataset(self, filenames: List, fps: int):
             )
             yield
 
-    def pull(self):
+    def pull(self, image_status: str = ""):
         """Downloads a rermote project (images and annotations) in the projects directory. """
-        response = self._client.get(
-            f"/datasets/{self.dataset_id}/export?format=json", raw=True
-        )
+        query = f"/datasets/{self.dataset_id}/export?format=json"
+        if image_status:
```

Comment:
I know this perfectly works fine, however, I like to avoid using empty string as False evaluators because it makes it somewhat less intuitive. For example, here we can use` image_status=None` in the signature and go ahead with a` if image_status is not None:` , what do you think?

Reply:
I totally agree. Changed it to `None`.